openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx

```diff
@@ -237,18 +235,9 @@ const AddAlertPage = () => {
       )?.join(', ')})`,
     }));
 
-    const modifiedAlertActions = alertActions?.map(
-      (action) =>
-        ({
-          ...action,
-          name: action.name ?? action.displayName,
-          displayName: action.displayName,
-        } as unknown as AlertAction)
-    );
-
     try {
       const requestAlertActions = await updateCreateAlertActions(
-        modifiedAlertActions
+        alertActions as unknown as AlertAction[]
```

Comment:
Do we need this type casting?

Reply:
Yes it's bit tricky, alertAction within Alert is EntityReference but when we try to create we need AlertAction